Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -501,7 +501,7 @@ private void DoBuild(string commands)
             Encoding encoding;
             try {
                 encoding = Encoding.GetEncoding(CultureInfo.CurrentCulture.TextInfo.OEMCodePage);
-            } catch (NotSupportedException) {
+            } catch (Exception ex) when (ex is NotSupportedException || ex is ArgumentException) {
```

Comment:
MonoGame is using an older version of C# (C# 5?), so this will have to handle both exceptions in separate catch blocks.

Reply:
Yeah, I asked @Jjagg if there are any plans to update the compiler, also given that there is movement towards .net standard. But I am also fine going the old ways.

Reply:
I removed the usage of the pattern matching.
Also I've found out that I was using two different github accounts. My apologies for the confusion.